playbooks/roles/kibana/templates/kibana.yml.j2

```diff
@@ -1,2 +1,5 @@
 server:
   name: {{ kibana_server_name }}
+
+elasticsearch.password: {{ kibana_elasticsearch_password }}
+xpack.encryptedSavedObjects.encryptionKey: {{ kibana_encryption_key }}
```

Comment:
Does this PR include [enabling TLS between Kibana and Elasticsearch](https://www.elastic.co/guide/en/kibana/7.9/configuring-tls.html#configuring-tls-kib-es)? This is also [listed as a requirement for using alerts](https://www.elastic.co/guide/en/kibana/7.9/alert-action-settings-kb.html#general-alert-action-settings). Some of the steps like preparing the certificates will need to be handled manually, but we'll need to install the appropriate files and set the required options here as well.

Reply:
@kaizoku  Added a new variable ``elasticsearch_pem_file`` which should be set to the pem file location of elasticsearch CA. Would that be ok? or we are going to use another option?